modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowForm.tsx

```diff
@@ -167,8 +186,10 @@ const TestflowForm = () => {
 	};
 
 	useEffect(() => {
-		setUsers(taskUser);
-	}, [setUsers, taskUser]);
+		if (testrayTaskUser) {
+			setUsers(testrayTaskUser.map(({user}) => user?.id as number));
```

Comment:
```suggestion
			setUsersId(testrayTaskUser.map(({user}) => user?.id as number));
```

ouldn't it be better ¨setUsersId¨

Reply:
Agree